Pipfile

```diff
@@ -26,7 +26,7 @@ pylint = "==3.0.0-a4"
 pytest = ">6"
 mypy = "0.790"
 ptvsd = "4.3.2"
-black = ">21"
+black = "==22.8.0"
```

Comment:
Why did you pin it to a particular version? Why not `>22.8`?

Reply:
Because reproducibility 🤷

Reply:
I am not sure they won't change formatting in patch version

Reply:
Fair enough